Modules/SubModule_ASL/xASL_wrp_Quantify.m

```diff
@@ -1,19 +1,20 @@
-function xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath, bUseBasilQuantification)
+function x = xASL_wrp_Quantify(x, PWI_Path, OutputPath, M0Path, SliceGradientPath, bUseBasilQuantification)
```

Comment:
Why the `x` output? This quantify wrapper is a submodule of `xASL_module_ASL`, and its outputs are quantified CBF NIfTI files, not `x`.

Reply:
in some cases, x-struct is modified to save certain variables for QC etc. But here, you have to be supercareful. This is for example connected with the bUseBasilQuantification. In the main xASL-module, this function is called once with BASIL=1 and once with BASIL=0. If you save it to x-struct and return, this will totally mess up things.

So bUseBasilQuantification should stay local, and you should prove if there's anything else saved in the x-struct that needs to be passed as an output.